portability-core/src/main/java/org/dataportabilityproject/serviceProviders/google/contacts/GoogleContactsService.java

```diff
@@ -152,22 +168,25 @@ private static StructuredName convertSinglePersonName(Name personName) {
 
   @Override
   public ContactsModelWrapper export(ExportInformation continuationInformation) throws IOException {
-    // TODO(olsona): implement pagination
-
-    Collection<VCard> vCards = new HashSet<>();
-
-    List<String> resourceNames = peopleService.contactGroups().batchGet().getResourceNames();
-    for (String resourceName : resourceNames) {
-      Person person = peopleService.people().get(resourceName).execute();
-      VCard vcard = convertToModel(person);
-      vCards.add(vcard);
+    // TODO(olsona): what about the next page?  We get the token, but how do we use it?
```

Comment:
It looks like you can pass the 'pageToken' query param. Do you want to convert this to a TODO to use that?
See:
https://developers.google.com/people/api/rest/v1/people.connections/list